tests/suites/0_stateless/03_0010_select_in_operator.sql

```diff
@@ -0,0 +1,3 @@
+select number from numbers_mt(10) where number+1 in (2, 3 ,20) order by number;
+select number from numbers_mt(10000) where number+1 in (2, 3 ,20, 5000) order by number;
+select number from numbers_mt(10) where number+1 not in (2, 3 ,20, 5000) order by number;
```

Comment:
Seems the code not work in list with null, should add a null case, like ` in (2, null,3)`

Reply:
Sure. Will do.